src/test/java/uk/ac/cam/cl/dtg/isaac/api/managers/GameManagerTest.java

```diff
@@ -45,24 +44,25 @@
 import uk.ac.cam.cl.dtg.segue.dao.content.ContentManagerException;
 import uk.ac.cam.cl.dtg.segue.dao.content.GitContentManager;
 import uk.ac.cam.cl.dtg.segue.dao.content.GitContentManager.BooleanSearchClause;
+import uk.ac.cam.cl.dtg.util.mappers.ContentMapper;
 
 class GameManagerTest {
   private GitContentManager dummyContentManager;
   private GameboardPersistenceManager dummyGameboardPersistenceManager;
-  private MapperFacade dummyMapper;
+  private ContentMapper objectMapper;
   private QuestionManager dummyQuestionManager;
   private GameManager gameManager;
 
   @BeforeEach
   public void setUp() {
     this.dummyContentManager = createMock(GitContentManager.class);
     this.dummyGameboardPersistenceManager = createMock(GameboardPersistenceManager.class);
-    this.dummyMapper = createMock(MapperFacade.class);
+    this.objectMapper = ContentMapper.INSTANCE;
```

Comment:
Why are we passing an actual mapper now, instead of a mock? Is this actually needed?